src/core/client/stream/tabs/Comments/Stream/AllCommentsTab/CommentEnteredSubscription.tsx

```diff
@@ -175,6 +175,17 @@ const CommentEnteredSubscription = createSubscription(
               tags {
                 code
               }
+              story {
+                commentCounts {
+                  totalPublished
+                  tags {
+                    FEATURED
+                    UNANSWERED
+                    REVIEW
+                    QUESTION
+                  }
+                }
+              }
```

Comment:
This won't fix the count issues, rather it conflicts with other count logic. You should look to update `AllCommentsTabViewNewMutation` instead.